include/zephyr/dt-bindings/pinctrl/numaker_m46x_pinctrl.h

```diff
@@ -0,0 +1,2154 @@
+#define SYS_GPG_MFP0_PG1MFP_Pos          (8)
+#define SYS_GPG_MFP0_PG2MFP_Pos          (16)
+#define SYS_GPG_MFP0_PG3MFP_Pos          (24)
+#define SYS_GPG_MFP1_PG4MFP_Pos          (0)
+#define SYS_GPG_MFP1_PG5MFP_Pos          (8)
+#define SYS_GPG_MFP1_PG6MFP_Pos          (16)
+#define SYS_GPG_MFP1_PG7MFP_Pos          (24)
+#define SYS_GPG_MFP2_PG8MFP_Pos          (0)
+#define SYS_GPG_MFP2_PG9MFP_Pos          (8)
+#define SYS_GPG_MFP2_PG10MFP_Pos         (16)
+#define SYS_GPG_MFP2_PG11MFP_Pos         (24)
+#define SYS_GPG_MFP3_PG12MFP_Pos         (0)
+#define SYS_GPG_MFP3_PG13MFP_Pos         (8)
+#define SYS_GPG_MFP3_PG14MFP_Pos         (16)
+#define SYS_GPG_MFP3_PG15MFP_Pos         (24)
+
+#define SYS_GPH_MFP0_PH0MFP_Pos          (0)
+#define SYS_GPH_MFP0_PH1MFP_Pos          (8)
+#define SYS_GPH_MFP0_PH2MFP_Pos          (16)
+#define SYS_GPH_MFP0_PH3MFP_Pos          (24)
+#define SYS_GPH_MFP1_PH4MFP_Pos          (0)
+#define SYS_GPH_MFP1_PH5MFP_Pos          (8)
+#define SYS_GPH_MFP1_PH6MFP_Pos          (16)
+#define SYS_GPH_MFP1_PH7MFP_Pos          (24)
+#define SYS_GPH_MFP2_PH8MFP_Pos          (0)
+#define SYS_GPH_MFP2_PH9MFP_Pos          (8)
+#define SYS_GPH_MFP2_PH10MFP_Pos         (16)
+#define SYS_GPH_MFP2_PH11MFP_Pos         (24)
+#define SYS_GPH_MFP3_PH12MFP_Pos         (0)
+#define SYS_GPH_MFP3_PH13MFP_Pos         (8)
+#define SYS_GPH_MFP3_PH14MFP_Pos         (16)
+#define SYS_GPH_MFP3_PH15MFP_Pos         (24)
+
+#define SYS_GPI_MFP1_PI6MFP_Pos          (16)
+#define SYS_GPI_MFP1_PI7MFP_Pos          (24)
+#define SYS_GPI_MFP2_PI8MFP_Pos          (0)
+#define SYS_GPI_MFP2_PI9MFP_Pos          (8)
+#define SYS_GPI_MFP2_PI10MFP_Pos         (16)
+#define SYS_GPI_MFP2_PI11MFP_Pos         (24)
+#define SYS_GPI_MFP3_PI12MFP_Pos         (0)
+#define SYS_GPI_MFP3_PI13MFP_Pos         (8)
+#define SYS_GPI_MFP3_PI14MFP_Pos         (16)
+#define SYS_GPI_MFP3_PI15MFP_Pos         (24)
+
+#define SYS_GPJ_MFP0_PJ0MFP_Pos          (0)
+#define SYS_GPJ_MFP0_PJ1MFP_Pos          (8)
+#define SYS_GPJ_MFP0_PJ2MFP_Pos          (16)
+#define SYS_GPJ_MFP0_PJ3MFP_Pos          (24)
+#define SYS_GPJ_MFP1_PJ4MFP_Pos          (0)
+#define SYS_GPJ_MFP1_PJ5MFP_Pos          (8)
+#define SYS_GPJ_MFP1_PJ6MFP_Pos          (16)
+#define SYS_GPJ_MFP1_PJ7MFP_Pos          (24)
+#define SYS_GPJ_MFP2_PJ8MFP_Pos          (0)
+#define SYS_GPJ_MFP2_PJ9MFP_Pos          (8)
+#define SYS_GPJ_MFP2_PJ10MFP_Pos         (16)
+#define SYS_GPJ_MFP2_PJ11MFP_Pos         (24)
+#define SYS_GPJ_MFP3_PJ12MFP_Pos         (0)
+#define SYS_GPJ_MFP3_PJ13MFP_Pos         (8)
+
+/* End of M460 BSP sys_reg.h pin-mux module copy */
+
+/* Beginning of M460 BSP sys.h pin-mux module copy */
+
+/*
+ *----------------------------------------------------------------------------
+ *  Multi-Function constant definitions.
+ *----------------------------------------------------------------------------
+ */
+
+/* PA.0 MFP */
+#define SYS_GPA_MFP0_PA0MFP_GPIO         (0x00UL<<SYS_GPA_MFP0_PA0MFP_Pos)
```

Comment:
please format this file properly, e.g. this is missing spaces around << (try clang-format)